src/osd/scheduler/mClockScheduler.cc

```diff
@@ -398,6 +432,10 @@ void mClockScheduler::enqueue(OpSchedulerItem&& item)
     immediate.push_front(std::move(item));
   } else {
     int cost = calc_scaled_cost(item.get_cost());
+    dout(15) << __func__ << " " << id
```

Comment:
nit: how about dout(20)? 

In general, I think it makes sense to keep things at level 1,10 or 20 for mclock logging. WDYT?

Reply:
Sure, I had set it to 15 since the corresponding log level in OSD::enqueue_op is 15. But now since the debug_mclock subsystem is introduced, we can use 1, 10, 20 or 30.